index.js

```diff
@@ -62,6 +84,14 @@ function $asNull () {
   return 'null'
 }
 
+function $asInteger (i) {
+  if (isLong && isLong(i)) {
+    return i.toString()
+  } else {
+    return $asNumber(i)
+  }
+}
```

Comment:
those ifs are really not good for performance if long is not in the dependency tree. We should use two predeclared functions, and select one base on long availability.